letsdrill.rb

```diff
@@ -1,24 +1,37 @@
 def get_letter_grade(integer)
- 
+
   #Put your code here!
+  case integer
+  when 90..100
+    return "A"
```

Comment:
Thanks for the suggestion, the implicit return is pretty convenient.

On Mon, Nov 16, 2015 at 12:54 PM, Michael Gee <notifications@github.com>
wrote:

> In letsdrill.rb
> <https://github.com/paircolumbus/LetsDrill/pull/42#discussion_r44957196>:
>
> >    #Put your code here!
> > +  case integer
> > +  when 90..100
> > +    return "A"
>
> Rubyists tend to avoid explicit return. This method should still work
> without it. A la:
>
> when 90..100
>   "A"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paircolumbus/LetsDrill/pull/42/files#r44957196>.
>